The following directories, files and symlinks will be created:
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/azureiot
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/bear
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/bitgroomingz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/cusz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/cuszp
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/datastates_ai
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/dctz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/faz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/fzvis
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/halide
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/lc_framework
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libdistributed
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libice
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_adios1
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_adios2
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_dataset
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_errorinjector
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_frsz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_jit
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_nvcomp
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_opt
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_predict
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_rmetric
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_sperr
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_tools
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_tthresh
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libstdcompat
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libunifex
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/liburing
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/m
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mdspan
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mgard
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mpigdb
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/msz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/opentelemetry_cpp
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/pixman
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/poorjit
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_cf_xarray
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_langchain_community
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_langchain_core
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_langsmith
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_ldcpy
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_m
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_passlib
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_perftrace
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_pytorch_wavelets
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_tezip
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_tomocupy
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_xrft
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/qcat
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/qoz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/reeber
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/sperr
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/sz
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/sz3
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/szx
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/tezip
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/tif22pnm
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/z_checker_tools
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/zfp
create directory /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/szp
copy /home/runderwood/git/contrib/robertu94_packages/packages/szp/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/szp/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/zfp/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/zfp/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/z-checker-tools/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/z_checker_tools/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/tif22pnm/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/tif22pnm/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/tezip/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/tezip/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/szx/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/szx/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/sz3/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/sz3/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/sz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/sz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/sperr/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/sperr/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/reeber/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/reeber/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/reeber/install_examples.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/reeber/install_examples.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/qoz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/qoz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/qcat/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/qcat/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-xrft/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_xrft/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-tomocupy/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_tomocupy/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-tezip/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_tezip/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-pytorch-wavelets/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_pytorch_wavelets/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-perftrace/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_perftrace/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-passlib/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_passlib/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-m/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_m/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-ldcpy/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_ldcpy/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-langsmith/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_langsmith/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-langchain-core/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_langchain_core/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-langchain-community/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_langchain_community/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/py-cf-xarray/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/py_cf_xarray/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/poorjit/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/poorjit/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/pixman/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/pixman/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/pixman/clang.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/pixman/clang.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/opentelemetry-cpp/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/opentelemetry_cpp/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/msz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/msz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/mpigdb/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mpigdb/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/mgard/hip-cub-configure.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mgard/hip-cub-configure.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/mgard/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mgard/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/mgard/hip-pointer-attribute-struct-fix.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mgard/hip-pointer-attribute-struct-fix.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/mdspan/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/mdspan/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/liburing/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/liburing/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libunifex/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libunifex/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libstdcompat/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libstdcompat/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-tthresh/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_tthresh/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-tools/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_tools/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-sperr/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_sperr/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-rmetric/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_rmetric/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-predict/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_predict/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-opt/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_opt/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-nvcomp/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_nvcomp/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-jit/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_jit/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-frsz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_frsz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-errorinjector/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_errorinjector/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-dataset/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_dataset/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-adios2/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_adios2/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio-adios1/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio_adios1/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libpressio/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libpressio/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libice/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libice/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/libdistributed/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/libdistributed/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/lc-framework/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/lc_framework/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/fzvis/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/fzvis/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/faz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/faz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/dctz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/dctz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/datastates-ai/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/datastates_ai/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/cuszp/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/cuszp/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/cusz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/cusz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/bitgroomingz/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/bitgroomingz/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/bear/rpath-handling-3.0.20.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/bear/rpath-handling-3.0.20.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/bear/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/bear/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/package.py -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/package.py
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/2.10-enable-rocm6.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/2.10-enable-rocm6.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/2.7.1-fix-broken-endian-reverse-compile.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/2.7.1-fix-broken-endian-reverse-compile.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/2.9.2-cmake-find-threads-package-first.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/2.9.2-cmake-find-threads-package-first.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/2.7-fix-missing-cstdint-include.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/2.7-fix-missing-cstdint-include.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/2.5-fix-clear_cache.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/2.5-fix-clear_cache.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/2.6-fix-gcc10-symbols.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/2.6-fix-gcc10-symbols.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/2.7-fix-python-test-deps.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/2.7-fix-python-test-deps.patch
copy /home/runderwood/git/contrib/robertu94_packages/packages/adios2/cmake-update-findmpi.patch -> /home/runderwood/git/contrib/robertu94_packages/spack_repo/robertu94/packages/adios2/cmake-update-findmpi.patch

--- a/packages/adios2/package.py
+++ b/packages/adios2/package.py
@@ -6,6 +6,9 @@
 import os
 import tempfile
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.rocm import ROCmPackage
 from spack.package import *
 
 
--- a/packages/bear/package.py
+++ b/packages/bear/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/bitgroomingz/package.py
+++ b/packages/bitgroomingz/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/cusz/package.py
+++ b/packages/cusz/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.cusz.package import Cusz as BuiltinCusz
 except ImportError:
-    from spack.pkg.builtin.cusz import Cusz as BuiltinCusz
+    from spack_repo.builtin.packages.cusz.package import Cusz as BuiltinCusz
 
 
 class Cusz(BuiltinCusz):
--- a/packages/cuszp/package.py
+++ b/packages/cuszp/package.py
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
 from spack.package import *
 
 
--- a/packages/datastates-ai/package.py
+++ b/packages/datastates-ai/package.py
@@ -2,6 +2,7 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/dctz/package.py
+++ b/packages/dctz/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/faz/package.py
+++ b/packages/faz/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 class Faz(CMakePackage):
--- a/packages/fzvis/package.py
+++ b/packages/fzvis/package.py
@@ -2,6 +2,7 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.generic import Package
 from spack.package import *
 import shutil
 
--- a/packages/lc-framework/package.py
+++ b/packages/lc-framework/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.lc_framework.package  import LcFramework as BuiltinLcFramework
 except ImportError:
-    from spack.pkg.builtin.lc_framework import LcFramework as BuiltinLcFramework
+    from spack_repo.builtin.packages.lc_framework.package import LcFramework as BuiltinLcFramework
 
 class LcFramework(BuiltinLcFramework):
     """a framework for automatically creating high-speed lossless and error-bounded lossy data compression and decompression algorithms."""
--- a/packages/libdistributed/package.py
+++ b/packages/libdistributed/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libdistributed.package  import Libdistributed as LibdistributedBuiltin
 except ImportError:
-    from spack.pkg.builtin.libdistributed import Libdistributed as LibdistributedBuiltin
+    from spack_repo.builtin.packages.libdistributed.package import Libdistributed as LibdistributedBuiltin
 
 
 class Libdistributed(LibdistributedBuiltin):
--- a/packages/libice/package.py
+++ b/packages/libice/package.py
@@ -7,7 +7,7 @@
 try:
         from spack_repo.builtin.packages.libice.package  import Libice as BuiltinLibice
 except ImportError:
-    from spack.pkg.builtin.libice import Libice as BuiltinLibice
+    from spack_repo.builtin.packages.libice.package import Libice as BuiltinLibice
 
 class Libice(BuiltinLibice):
     """libICE - Inter-Client Exchange Library."""
--- a/packages/libpressio/package.py
+++ b/packages/libpressio/package.py
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
 from spack.package import *
 
 
--- a/packages/libpressio-adios1/package.py
+++ b/packages/libpressio-adios1/package.py
@@ -2,6 +2,7 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/libpressio-adios2/package.py
+++ b/packages/libpressio-adios2/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_adios2.package  import LibpressioAdios2 as BuiltinLibPressioAdios2
 except ImportError:
-    from spack.pkg.builtin.libpressio_adios2 import LibpressioAdios2 as BuiltinLibPressioAdios2
+    from spack_repo.builtin.packages.libpressio_adios2.package import LibpressioAdios2 as BuiltinLibPressioAdios2
 
 
 class LibpressioAdios2(BuiltinLibPressioAdios2):
--- a/packages/libpressio-dataset/package.py
+++ b/packages/libpressio-dataset/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/libpressio-errorinjector/package.py
+++ b/packages/libpressio-errorinjector/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_errorinjector.package  import LibpressioErrorinjector as BuiltinLibPressioErrorInjector
 except ImportError:
-    from spack.pkg.builtin.libpressio_errorinjector import LibpressioErrorinjector as BuiltinLibPressioErrorInjector
+    from spack_repo.builtin.packages.libpressio_errorinjector.package import LibpressioErrorinjector as BuiltinLibPressioErrorInjector
 
 
 class LibpressioErrorinjector(BuiltinLibPressioErrorInjector):
--- a/packages/libpressio-frsz/package.py
+++ b/packages/libpressio-frsz/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/libpressio-jit/package.py
+++ b/packages/libpressio-jit/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/libpressio-nvcomp/package.py
+++ b/packages/libpressio-nvcomp/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_nvcomp.package  import LibpressioNvcomp as BuiltinLibPressioNvcomp
 except ImportError:
-    from spack.pkg.builtin.libpressio_nvcomp import LibpressioNvcomp as BuiltinLibPressioNvcomp
+    from spack_repo.builtin.packages.libpressio_nvcomp.package import LibpressioNvcomp as BuiltinLibPressioNvcomp
 
 
 class LibpressioNvcomp(BuiltinLibPressioNvcomp):
--- a/packages/libpressio-opt/package.py
+++ b/packages/libpressio-opt/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_opt.package  import LibpressioOpt as BuiltinLibpressioOpt
 except ImportError:
-    from spack.pkg.builtin.libpressio_opt import LibpressioOpt as BuiltinLibpressioOpt
+    from spack_repo.builtin.packages.libpressio_opt.package import LibpressioOpt as BuiltinLibpressioOpt
 
 
 class LibpressioOpt(BuiltinLibpressioOpt):
--- a/packages/libpressio-predict/package.py
+++ b/packages/libpressio-predict/package.py
@@ -6,7 +6,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_predict.package  import LibpressioPredict as BuiltinLibpressioPredict
 except ImportError:
-    from spack.pkg.builtin.libpressio_predict import LibpressioPredict as BuiltinLibpressioPredict
+    from spack_repo.builtin.packages.libpressio_predict.package import LibpressioPredict as BuiltinLibpressioPredict
 
 
 class LibpressioPredict(BuiltinLibpressioPredict):
--- a/packages/libpressio-rmetric/package.py
+++ b/packages/libpressio-rmetric/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_rmetric.package  import LibpressioRmetric as BuiltinLibPressioRmetric
 except ImportError:
-    from spack.pkg.builtin.libpressio_rmetric import LibpressioRmetric as BuiltinLibPressioRmetric
+    from spack_repo.builtin.packages.libpressio_rmetric.package import LibpressioRmetric as BuiltinLibPressioRmetric
 
 
 class LibpressioRmetric(BuiltinLibPressioRmetric):
--- a/packages/libpressio-sperr/package.py
+++ b/packages/libpressio-sperr/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_sperr.package  import LibpressioSperr as BuiltinLibPressioSperr
 except ImportError:
-    from spack.pkg.builtin.libpressio_sperr import LibpressioSperr as BuiltinLibPressioSperr
+    from spack_repo.builtin.packages.libpressio_sperr.package import LibpressioSperr as BuiltinLibPressioSperr
 
 
 class LibpressioSperr(BuiltinLibPressioSperr):
--- a/packages/libpressio-tools/package.py
+++ b/packages/libpressio-tools/package.py
@@ -2,7 +2,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_tools.package  import LibpressioTools as BuiltinLibPressioTools
 except ImportError:
-    from spack.pkg.builtin.libpressio_tools import LibpressioTools as BuiltinLibPressioTools
+    from spack_repo.builtin.packages.libpressio_tools.package import LibpressioTools as BuiltinLibPressioTools
 
 
 class LibpressioTools(BuiltinLibPressioTools):
--- a/packages/libpressio-tthresh/package.py
+++ b/packages/libpressio-tthresh/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libpressio_tthresh.package  import LibpressioTthresh as BuiltinLibPressioTthresh
 except ImportError:
-    from spack.pkg.builtin.libpressio_tthresh import LibpressioTthresh as BuiltinLibPressioTthresh
+    from spack_repo.builtin.packages.libpressio_tthresh.package import LibpressioTthresh as BuiltinLibPressioTthresh
 
 
 class LibpressioTthresh(BuiltinLibPressioTthresh):
--- a/packages/libstdcompat/package.py
+++ b/packages/libstdcompat/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.libstdcompat.package  import Libstdcompat as BuiltinStdCompat
 except ImportError:
-    from spack.pkg.builtin.libstdcompat import Libstdcompat as BuiltinStdCompat
+    from spack_repo.builtin.packages.libstdcompat.package import Libstdcompat as BuiltinStdCompat
 
 
 class Libstdcompat(BuiltinStdCompat):
--- a/packages/libunifex/package.py
+++ b/packages/libunifex/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/liburing/package.py
+++ b/packages/liburing/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.autotools import AutotoolsPackage
 from spack.package import *
 
 
--- a/packages/mdspan/package.py
+++ b/packages/mdspan/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/mgard/package.py
+++ b/packages/mgard/package.py
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.build_systems.cmake import CMakeBuilder
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.rocm import ROCmPackage
 from spack.package import *
 
 
--- a/packages/mpigdb/package.py
+++ b/packages/mpigdb/package.py
@@ -2,6 +2,7 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.generic import Package
 from spack.package import *
 
 
--- a/packages/msz/package.py
+++ b/packages/msz/package.py
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
 from spack.package import *
 
 
--- a/packages/opentelemetry-cpp/package.py
+++ b/packages/opentelemetry-cpp/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/pixman/package.py
+++ b/packages/pixman/package.py
@@ -5,6 +5,7 @@
 
 import sys
 
+from spack_repo.builtin.build_systems.autotools import AutotoolsPackage
 from spack.package import *
 import spack
 
--- a/packages/poorjit/package.py
+++ b/packages/poorjit/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/py-cf-xarray/package.py
+++ b/packages/py-cf-xarray/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-langchain-community/package.py
+++ b/packages/py-langchain-community/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-langchain-core/package.py
+++ b/packages/py-langchain-core/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-langsmith/package.py
+++ b/packages/py-langsmith/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-ldcpy/package.py
+++ b/packages/py-ldcpy/package.py
@@ -19,6 +19,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-m/package.py
+++ b/packages/py-m/package.py
@@ -1,3 +1,4 @@
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-passlib/package.py
+++ b/packages/py-passlib/package.py
@@ -19,6 +19,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-perftrace/package.py
+++ b/packages/py-perftrace/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.generic import Package
 from spack.package import *
 
 
--- a/packages/py-pytorch-wavelets/package.py
+++ b/packages/py-pytorch-wavelets/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-tezip/package.py
+++ b/packages/py-tezip/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-tomocupy/package.py
+++ b/packages/py-tomocupy/package.py
@@ -2,6 +2,7 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/py-xrft/package.py
+++ b/packages/py-xrft/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.python import PythonPackage
 from spack.package import *
 
 
--- a/packages/qcat/package.py
+++ b/packages/qcat/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/qoz/package.py
+++ b/packages/qoz/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.qoz.package  import Qoz as BuiltinQoz
 except ImportError:
-    from spack.pkg.builtin.qoz import Qoz as BuiltinQoz
+    from spack_repo.builtin.packages.qoz.package import Qoz as BuiltinQoz
 
 
 class Qoz(BuiltinQoz):
--- a/packages/reeber/package.py
+++ b/packages/reeber/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/sperr/package.py
+++ b/packages/sperr/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/sz/package.py
+++ b/packages/sz/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.sz.package  import Sz as SzBuiltin, CMakeBuilder as SZCMakeBuilder
 except ImportError:
-    from spack.pkg.builtin.sz import Sz as SzBuiltin, SZCMakeBuilder
+    from spack_repo.builtin.packages.sz.package import Sz as SzBuiltin, SZCMakeBuilder
 
 
 class Sz(SzBuiltin):
--- a/packages/sz3/package.py
+++ b/packages/sz3/package.py
@@ -7,7 +7,7 @@
 try:
     from spack_repo.builtin.packages.sz3.package  import Sz3 as Sz3Builtin
 except ImportError:
-    from spack.pkg.builtin.sz3 import Sz3 as Sz3Builtin
+    from spack_repo.builtin.packages.sz3.package import Sz3 as Sz3Builtin
 
 
 class Sz3(Sz3Builtin):
--- a/packages/szx/package.py
+++ b/packages/szx/package.py
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 import spack.build_systems.autotools
 import spack.build_systems.cmake
+from spack_repo.builtin.build_systems.autotools import AutotoolsPackage
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
 from spack.package import *
 
 
--- a/packages/tezip/package.py
+++ b/packages/tezip/package.py
@@ -1,6 +1,8 @@
 # Copyright Spack Project Developers. See COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
 from spack.package import *
 
 
--- a/packages/tif22pnm/package.py
+++ b/packages/tif22pnm/package.py
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.generic import Package
 from spack.package import *
 
 
--- a/packages/z-checker-tools/package.py
+++ b/packages/z-checker-tools/package.py
@@ -20,6 +20,7 @@
 # See the Spack documentation for more information on packaging.
 # ----------------------------------------------------------------------------
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
--- a/packages/zfp/package.py
+++ b/packages/zfp/package.py
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
 from spack.package import *
 
 
--- a/packages/szp/package.py
+++ b/packages/szp/package.py
@@ -1,6 +1,7 @@
 # Copyright Spack Project Developers. See COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.cmake import CMakePackage
 from spack.package import *
 
 
